app/Module/AbstractIndividualListModule.php

```diff
@@ -383,7 +383,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
 
                         if ($surname !== '' || $show_all === 'yes') {
                             if ($show_all !== 'yes') {
```

Comment:
If we are changing this, then should we also remove the `if { .. }` around it?

When showing all surnames, we also want to have this new title?

Reply:
{ } of foreach seems not to be possible,